core/controllers/base_test.py

```diff
@@ -1661,17 +1661,18 @@ def test_handler_with_schemas_should_not_be_present_in_schema_requiring_handler_
         in payload validator only contains handler class names which require
         schemas.
         """
+
         list_of_handlers_to_be_removed = []
-        handler_names_which_do_not_require_schemas = (
-            payload_validator.HANDLER_CLASS_NAMES_WHICH_DO_NOT_REQUIRE_SCHEMAS)
+        handler_names_which_require_schemas = (
+            payload_validator.SCHEMA_REQUIRING_HANDLER_CLASS_NAMES)
```

Comment:
HANDLER_CLASS_NAMES_WHICH_STILL_NEED_SCHEMAS is probably clearer and more in line with some of the other names.

Reply:
Done